README.md

```diff
@@ -550,6 +550,24 @@ Expressions should finish quickly or the user experience may drag, especially on
 - [quick-execution](https://angular.io/guide/template-syntax#quick-execution) - official documentation for template expressions
 - [Increasing Performance - more than a pipe dream](https://youtu.be/I6ZvpdRM1eQ) - ng-conf video on youtube. Using pipe instead of function in interpolation expression
 
+### Coalescing event change detections
+
+Angular uses ZoneJS to intercept events that occurred in the application and run a change detection cycle automatically, also Angular runs change detection every time when the microtask queue is empty.
+From V9, Angular provides a way to coalesce event change detections by turning `ngZoneEventCoalescing` on, i.e
+```
+platformBrowser()
+  .bootstrapModule(AppModule, { ngZoneEventCoalescing: true });
+```
+which will schedule running change detection with `requestAnimationFrame`.
```

Comment:
```suggestion
The above configuration will schedule change detection with `requestAnimationFrame`, instead of plugging into the microtask queue, which will run checks less frequently and consume fewer computational cycles.
```